cli/batch-score-rest.sh

```diff
@@ -0,0 +1,311 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+# <create_variables>
+SUBSCRIPTION_ID="5f08d643-1910-4a38-a7c7-84a39d4f42e0"
+LOCATION="centraluseuap"
+RESOURCE_GROUP="chpirillrg"
+TENANT_ID=
+
+WORKSPACE="chpirillcentral"
+API_VERSION="2021-10-01"
+
+TOKEN=$(az account get-access-token --query accessToken -o tsv)
```

Comment:
TOKEN=$(az account get-access-token --resource https://ml.azure.com --query accessToken -o tsv)

Reply:
`--resource https://ml.azure.com` does not provide access to the ARM resources, only to the ScoringUri.

A fully scoped token is required for all of the ARM calls.